cli/UploadSourcemaps.ts

```diff
@@ -0,0 +1,88 @@
+import axios from 'axios';
+import { Command, Option } from 'commander';
+import FormData from 'form-data';
+import fs from 'fs';
+import path from 'path';
+
+export const uploadSourcemapsCommand: Command = new Command();
+
+uploadSourcemapsCommand
+  .name('upload-sourcemaps')
+  .addOption(
+    new Option('-p, --platform <value>', 'Platform')
+      .choices(['ios', 'android'])
+      .makeOptionMandatory(),
+  )
+  .addOption(
+    new Option(
+      '-d, --dir <value>',
+      'The path of the directory including the source map file',
+    ).makeOptionMandatory(),
+  )
+  .addOption(
+    new Option('-t, --token <value>', 'Your App Token')
+      .env('INSTABUG_APP_TOKEN')
+      .makeOptionMandatory(),
+  )
+  .addOption(
+    new Option('-v, --name <value>', 'The app version name')
+      .env('INSTABUG_APP_VERSION_NAME')
+      .makeOptionMandatory(),
+  )
+  .addOption(
+    new Option('-c, --code <value>', 'The app version code')
+      .env('INSTABUG_APP_VERSION_CODE')
+      .makeOptionMandatory(),
+  )
+  .addOption(new Option('-l, --label <value>', "The CodePush label if it's a CodePush release"))
+  .action(function (this: Command) {
+    const options = this.opts<UploadSourcemapsOptions>();
+    uploadFile(options);
+  })
+  .showHelpAfterError();
+
+interface UploadSourcemapsOptions {
+  platform: 'android' | 'ios';
+  dir: string;
+  token: string;
+  name: string;
+  code: string;
+  label?: string;
+}
+
+const uploadFile = async (options: UploadSourcemapsOptions) => {
+  const os = options.platform;
+  const dir = options.dir;
+  const token = options.token;
+  const name = options.name;
+  const code = options.code;
+  const label = options.label;
+  const version = {
+    code: code,
+    name: name,
+    codePush: label,
+  };
+  const form = new FormData();
+  form.append('app_version', JSON.stringify(version));
+  form.append(
+    'symbols_file',
+    fs.readFileSync(path.join(dir, `${os}-sourcemap.json`)),
+    `${os}-sourcemap.json`,
+  );
+  form.append('application_token', token);
+  form.append('platform', 'react_native');
+  form.append('os', os);
```

Comment:
**suggestion:** This might contains multiple comments, just to show the final suggestion:

1. A more descriptive function name can be `uploadFile`.
1. It might be a matter of taste, but I think the re-assigning of `options` might be pretty verbose. I would suggest using `opts` for short and more readable inline code. What do you think?
1. Using blank lines between related lines of code can improve readability.
1. Extracting varaibles related to variables might be more readable as well IMO, instead of having a compound line. A rule of thumb I use is _"Never call a function inside the argument of another function"_. 

```suggestion
const uploadSourcemaps = async (opts: UploadSourcemapsOptions) => {
  const fileName = `${opts.platform}-sourcemap.json`;
  const filePath = path.join(opts.dir, fileName);
  const fileBlob = fs.readFileSync(filePath);

  const version = {
    code: opts.code,
    name: opts.name,
    codePush: opts.code,
  };

  const form = new FormData();
  form.append('platform', 'react_native');
  form.append('application_token', opts.token);
  form.append('os', opts.platform);
  form.append('symbols_file', fileBlob, fileName);
  form.append('app_version', JSON.stringify(version));
```

Reply:
I agree that all the aforementioned points positively impact readability. 👏 💡